rapid7_insight_agent/help.md

```diff
@@ -0,0 +1,173 @@
+# Description
+
+Using the power of the [Insight Agent](https://docs.rapid7.com/insight-agent/overview/) investigate and mitigate vulnerabilities across your organization. 
+
+# Key Features
+
+* Get details about devices
+* Qurantine and unquarantine devices
+
+# Requirements
+
+* Platform API Key: [Mange Platform API Keys](https://docs.rapid7.com/insight/managing-platform-api-keys/)
+* Organization ID: (See Page 10 of the GraphQL API doc)
```

Comment:
Can you link the page here?

Reply:
I don' think the GraphQL API doc is released to customers at this point. We should document more here if we can do it with brevity

Reply:
I didn't know what to do with this...getting the org ID is rough. I had to pull it from the network traffic in chrome.

Reply:
Is this Org ID different the one in the browser URL when you log into the platform?

e..g https://us.connect.insight.rapid7.com/**ABCDEFGHIJKLMNOP123**#/home